core/src/cats/postgresql.cc

```diff
@@ -634,7 +634,7 @@ bool BareosDbPostgresql::SqlQueryWithoutHandler(const char* query, int flags)
       retval = true;
       break;
     case PGRES_FATAL_ERROR:
-      Dmsg1(50, "Result status fatal: %s\n", query);
+      Dmsg1(50, "Result status fatal: %s, %s\n", query, sql_strerror());
       if (exit_on_fatal_) { Emsg0(M_ERROR_TERM, 0, "Fatal database error\n"); }
```

Comment:
Quick question : shouldn't we also print the root cause if we exit_on_fatal__ so the exit cause is obvious even without debug level ?

Reply:
Makes sense, good idea!